src/Gui/QSint/actionpanel/actionpanel.cpp

```diff
@@ -11,110 +11,84 @@
 
 #include <QVariant>
 
-
 namespace QSint
 {
 
-
 ActionPanel::ActionPanel(QWidget *parent) :
     BaseClass(parent), mySpacer(nullptr)
 {
     setProperty("class", "panel");
-
     setScheme(ActionPanelScheme::defaultScheme());
-
     setSizePolicy(QSizePolicy::Maximum, QSizePolicy::Preferred);
 
-    QVBoxLayout *vbl = new QVBoxLayout();
+    auto *vbl = new QVBoxLayout();
     vbl->setContentsMargins(4, 8, 4, 8);
     vbl->setSpacing(8);
     setLayout(vbl);
 }
 
 void ActionPanel::setScheme(ActionPanelScheme *scheme)
 {
-  if (scheme) {
+    if (!scheme) return;
+
     myScheme = scheme;
     setStyleSheet(myScheme->actionStyle);
 
-    // set scheme for children
-    QObjectList list(children());
-    Q_FOREACH(QObject *obj, list) {
-      if (dynamic_cast<ActionGroup*>(obj)) {
-        ((ActionGroup*)obj)->setScheme(scheme);
-        continue;
-      }
-    }
-
     update();
-  }
 }
 
-//void ActionPanel::paintEvent ( QPaintEvent * event )
-//{
-//  //QPainter p(this);
-
-//  //p.setOpacity(0.5);
-//  //p.fillRect(rect(), myScheme->panelBackground);
-//}
-
 void ActionPanel::addWidget(QWidget *w)
 {
-  if (w)
-    layout()->addWidget(w);
+    if (w) layout()->addWidget(w);
 }
```

Comment:
I'm personally in favor of this, though there have been consensus that
```
if (w) {
    layout()->addWidget(w);
}
```
should always be used